handlers/SuiChainHandler.ts

```
@@ -644,4 +714,117 @@ export class SuiChainHandler extends BaseChainHandler<SuiChainConfig> {
+            // Additional validation: check if this might be for our deposit
+            // You could add more validation here based on timing, amount, etc.
+            logger.info(`Found potential transfer sequence ${parsedLog.args.transferSequence} in tx ${log.transactionHash}`);
+            
+            return {
+              sequence: parsedLog.args.transferSequence.toString(),
+              txHash: log.transactionHash,
+            };
+          }
+        } catch (error) {
+          continue;
+        }
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)